plugins/bufbuild/es/v1.3.0/Dockerfile

```diff
@@ -1,11 +1,21 @@
-# syntax=docker/dockerfile:1.4
-FROM node:18.16.1-alpine3.18 AS build
+# syntax=docker/dockerfile:1.7
+FROM node:18.20.3-bookworm AS build
```

Comment:
Kept the plugins that were on Node 18 on the same version but bumped images to latest security/bugfix releases.